src/bindings/python/src/pyopenvino/core/tensor.cpp

```diff
@@ -133,6 +133,14 @@ void regclass_Tensor(py::module m) {
             py::arg("type"),
             py::arg("shape"));
 
+    cls.def(py::init<ov::Output<ov::Node>>(), py::arg("node_port"));
+
+    cls.def(py::init([](ov::Output<ov::Node>& port, py::array& array) {
+                return ov::Tensor(port, const_cast<void*>(array.data(0)));
+            }),
+            py::arg("node_port"),
+            py::arg("array"));
```

Comment:
```suggestion
    cls.def(py::init<const ov::Output<const ov::Node>>(), py::arg("node_port"));

    cls.def(py::init([](const ov::Output<const ov::Node>& port, py::array& array) {
                return ov::Tensor(port, const_cast<void*>(array.data(0)));
            }),
            py::arg("node_port"),
            py::arg("array"));
```

There is a difference with classes `Output<const Node>` and `Output<Node>`. The outside qualifier is less (but still) important. However, there is significant difference with `<const Node>`, it comes usually from `CompiledModel`. The second `Ouput<Node>` comes usually from `Model`.

So according to:
https://github.com/openvinotoolkit/openvino/blob/3330cd69a7496ede1a55ec3feee649b8b8c8687c/src/core/include/openvino/core/node_output.hpp#L118
https://github.com/openvinotoolkit/openvino/blob/3330cd69a7496ede1a55ec3feee649b8b8c8687c/src/core/include/openvino/runtime/tensor.hpp#L146
https://github.com/openvinotoolkit/openvino/blob/3330cd69a7496ede1a55ec3feee649b8b8c8687c/src/core/include/openvino/runtime/tensor.hpp#L156

I believe that both constructors need ports from compiled models to work with. So this is required change for the constructors.

Reply:
After discussion, let's keep both const and non-const constructors.

Reply:
added constructors with const-output argument